external/runtime/runtime.depproj

```diff
@@ -133,5 +134,14 @@
       <ReferenceCopyLocalPaths Remove="@(ReferenceCopyLocalPathsNativeImages)" />
     </ItemGroup>
 
+    <!-- For coverage runs of any crossgen dll from CoreClr we need the respective IL, the pdbs are already in place -->
+    <Copy SourceFiles="@(CoreCLRILFiles)"
```

Comment:
You can just include your item in the `ReferenceCopyLocalPaths` and our bin place infrastructure will take care of doing the right thing to bin place the files. Just add it in line: https://github.com/dotnet/corefx/pull/30304/files#diff-2c6964675d4545109b9addd11497c910R127 like:

```xml
<ReferenceCopyLocalPaths Inclue="@(CoreCLRFiles);@(CoreCLRPDBFiles);@(CoreCLRILFiles)" />
```

Reply:
The problem is that the native image of CoreLib has the same name as the IL so the last one copied overwrites what was in @(CoreCLRFiles)

Reply:
Binplace support for sub-paths: It doesn't today but it probably should.  You could define a separate bin place configuration that takes in the IL items as input and bin places to the IL subfolder.  That's marginally simpler since you don't have to write a target.

Reply:
Copy seems straight forward and symmetric for il and ni folders, I'm leaning to keep it, unless there is a strong reason to change it.

Reply:
Whenever you restore the package you'll end up having this duplicate effort of copying or calling a common target that uses this copy. I think using the binplace configuration will be simpler as you'll only need to define the item and then specify it in the bin place configuration -- example: https://github.com/dotnet/corefx/blob/master/external/netfx/netfx.depproj#L74

There it defines and item, and previously it defines a bin place configuration with the correct target path and defining to bin place the contents of this item here:

https://github.com/dotnet/corefx/blob/master/external/netfx/netfx.depproj#L43-L46

It is straight forward as well and I think will be much cleaner whenever you start restoring the private package.

Reply:
I see, you're looking at the next step (getting the il from package) to get coverage back in CI